integration-tests/test_pulp.py

```diff
@@ -28,7 +28,10 @@
 
 from atomic_reactor.inner import DockerBuildWorkflow
 from atomic_reactor.util import ImageName
+# TODO: PulpUploader doesn't exist, remove or fix this test?
```

Comment:
Oh my... we should probably just get rid of this folder completely. @twaugh ?

Reply:
Yes, let's remove it.

Reply:
Feel free to open another PR to remove it.